src/ensembl/src/shared/components/show-hide/ShowHide.scss

```diff
@@ -0,0 +1,11 @@
+@import 'src/styles/common';
+
+.label {
+  color: $blue;
+  cursor: pointer;
+}
+
+.chevron {
+  margin-left: 10px;
+  height: 6px;
```

Comment:
Did we decide to reduce the size of the chevron here? 
![Screenshot 2021-07-01 at 12 00 03](https://user-images.githubusercontent.com/6347854/124113938-e7c50b80-da63-11eb-9d21-996e65a07d1b.png)

Reply:
Not sure I understand...